### ISSUE TITLE
Stunt maps

### ISSUE THREAD

**Issue by LimettenJunkie** [GreepTheSheep/openplanet-MXRandom#126]
**Describe the bug**
I played Random Map Survival and got a Stunt map in the stunt mode. The plugin told me to change the map, but when I pressed the button, it broke the plugin. The next map was normal, but it still told me to change the map, and when I tried finishing it hoping it would fix the issue, it added time to the timer, but still didn't fix the problem. Also, the timer went up to 18 minutes instead of 15 minutes max.

**To Reproduce**
Not sure if this will work, but I guess if you get a stunt map and click change map it should work. Its also important to note that i finished the stunt map before clicking change map, because i thought maybe the plugin would automatically change the map or so. Maybe that was the cause of the issue.

**Expected behavior**
I thought it would just change the map and continue my run normally, but it permanently paused the timer, and there was only the "End run" button as well as the "change maps" button.

**Screenshots**
unfortunately, I didn't think of screenshotting it

**Openplanet (please complete the following information):**
 - Version: 2.5.2
 - Game: TM2020, Openplanet (not sure what i was supposed to put here)
 

**Additional context**

**Comment by GreepTheSheep** [comment]
@LimettenJunkie if you have the map, can you send me its TrackmaniaExchange link of this ?

Stunt maps are flagged with `TM_Stunt` map type, and should not be on the plugin as it filters only to `TM_Race` map type (as it should for all other types like `TM_Royal` or even `FlagRushArena`)

**Comment by LimettenJunkie** [comment]
i think it was the first stunt map of the stunt campaign. made by tona. https://trackmania.exchange/maps/179949/stuntisback   i think this one

**Comment by GreepTheSheep** [comment]
Yeah this one is classified as `TM_Stunt` so it shouldn't be taken by the plugin. We need more testings and proofs to see if it does again

**Comment by Fort-TM** [comment]
This was most likely an issue with the API (TMX or Xertrov's), since the plugin filters by map type. There hasn't been other reports of this issue happening, so it might have been a one-time thing. 

Feel free to reopen the issue if it happens again, preferably with a log attached
